framework/wcf/Extensibility/Transport/AzureQueueStorageClient/AzureQueueStorageClient/AzureQueueStorageChannelHelpers.cs

```diff
@@ -0,0 +1,124 @@
+﻿//----------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+//----------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Net;
+using System.Net.Sockets;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+
+namespace Microsoft.Samples.AzureQueueStorage
+{
+    /// <summary>
+    /// Collection of constants used by the AzureQueueStorage Channel classes
+    /// </summary>
+    static class AzureQueueStorageConstants
+    {
+        internal const string EventLogSourceName = "Microsoft.Samples.AzureQueueStorage";
+        internal const string Scheme = "soap.aqs";
```

Comment:
I presume this will be released as a nuget package, so Samples shouldn't be in the name.